lib/config.js

```diff
@@ -29,15 +36,16 @@ if (argv.config) {
   if (typeof argv.config === 'string') {
     argv.config = [argv.config];
   }
-  for (var i in argv.config) {
-    loader.add(path.resolve(argv.config[i]));
-  }
+
+  argv.config.forEach(function(val) {
+    loader.add(path.resolve(val));
+  });
 }
 
 var setOptions = {};
 var key = null;
 for (key in yargs.argv) {
-  if (yargs.argv[key] !== undefined) {
+  if (!_.isUndefined(yargs.argv[key])) {
```

Comment:
It's better to use native javascript `hasOwnProperty` here: 

```javascript
for (key in yargs.argv) {
  if (yargs.argv.hasOwnProperty(key)) {
    setOptions[key] = yargs.argv[key];
  }
}
```

Using `hasOwnProperty` ensures that the object you're looping has the property directly, and it wasn't inherited from its prototype chain somehow. This is how you'd typically perform a safe for loop in JS. From what I can tell, lodash is doing type checking (like the original code was doing), which is not what this loop test should be doing anyway.

Lastly, I rarely see a good reason to introduce a new dependency when native JS can function.